Buildings/Experimental/DHC/Examples/Cooling/CoolingSystem.mo

```diff
@@ -0,0 +1,184 @@
+within Buildings.Experimental.DHC.Examples.Cooling;
+model CoolingSystem
+  "Example to test the district cooling network model"
+  extends Modelica.Icons.Example;
+  package Medium=Buildings.Media.Water
+    "Medium model for water";
+  parameter Integer nBui=1
```

Comment:
@JingWang-CUB: I wonder if the example here should be 2 or 3 buildings rather than 1? It appears to me that everything is correctly vectorized such that `nBui` can be easily changed to 2 and the correct simulation results are still achieved, but I wanted to confirm before finalizing the review. Thanks.

Reply:
Changed `nBui` to 2.